omnibus/config/projects/inspec.rb

```diff
@@ -17,21 +17,52 @@
 #
 
 name 'inspec'
+friendly_name "Inspec"
 maintainer 'Chef Software, Inc <maintainers@chef.io>'
 homepage 'https://github.com/chef/inspec'
 
 license 'Apache-2.0'
 license_file '../LICENSE'
 
-# Defaults to C:/inspec on Windows
-# and /opt/inspec on all other platforms
-install_dir "#{default_root}/#{name}"
+# Defaults to C:/opscode/inspec on Windows
+# and /opt/inspec on all other platforms.
+if windows?
+  install_dir "#{default_root}/opscode/#{name}"
+else
+  install_dir "#{default_root}/#{name}"
+end
 
 build_version Omnibus::BuildVersion.semver
 build_iteration 1
 
 dependency 'preparation'
+dependency 'ruby'
+dependency 'rb-readline'
+
 dependency 'inspec'
 
+dependency 'gem-permissions'
+dependency 'shebang-cleanup'
+dependency 'openssl-customization'
+dependency 'version-manifest'
```

Comment:
This isn't needed on newer omnibus and is only provided for backcompat.  Omnibus will create the version manifest by defaullt.